.travis.yml

```diff
@@ -12,11 +11,13 @@ env:
   - VARNISH_VERSION=4.0
 
 matrix:
+  allow_failures:
+    # HHVM support for CURLOPT_PROTOCOLS was added recently but not yet
+    # available on Travis
+    - php: hhvm
   include:
-    - php: 5.5
-      env: VARNISH_VERSION=3.0
```

Comment:
do we completely drop varnish 3 support? otherwise we should keep this. maybe as a second --prefer-lowest run.